<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPushFile</class>
 <widget class="QDialog" name="DialogPushFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка файла</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="wFileDescription" native="true">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <widget class="QLineEdit" name="leFileName">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblFileName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Наименование</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblFileDescription">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Краткое описание</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="teFileDescription">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="wFileDescription_Auto" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <widget class="QLineEdit" name="leFileSize">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblFileSize">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Размер файла, байт</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDownloadDate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дата загрузки</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leDB_Index">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblDB_Index">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Идентификатор файла в БД</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dteDownloadDate">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
